tests/network/autofs_server.pm

```diff
@@ -0,0 +1,32 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Maintainer: Antonio Caristia <acaristia@suse.com>
+
+use base 'consoletest';
+use testapi;
+use lockapi;
+use utils qw(systemctl zypper_call);
+use strict;
+
+sub run {
+    select_console "root-console";
+    my $test_share_dir = "/tmp/nfs/server";
+    zypper_call('in nfs-kernel-server');
+    assert_script_run "mkdir -p $test_share_dir";
+    assert_script_run "echo It worked > $test_share_dir/file.txt";
+    assert_script_run "echo $test_share_dir *(ro) >> /etc/exports";
+    assert_script_run "cat /etc/exports";
+    systemctl 'start nfs-server';
```

Comment:
I am not sure if you just added this in your last changes or you had that already from the beginning. I suggest you use the `systemctl` method in all other cases where you call systemctl as well. Can you do that?